pkg/project/api/validation/validation.go

```diff
@@ -13,11 +13,13 @@ func ValidateProject(project *api.Project) errors.ValidationErrorList {
 	result := errors.ValidationErrorList{}
 	if len(project.Name) == 0 {
 		result = append(result, errors.NewFieldRequired("Name", project.Name))
-	} else if !util.IsDNS952Label(project.Name) {
-		result = append(result, errors.NewFieldInvalid("Name", project.Name, "does not conform to lower-cased dns952"))
+	} else if !util.IsDNSSubdomain(project.Name) {
+		result = append(result, errors.NewFieldInvalid("Name", project.Name, ""))
 	}
-	if !util.IsDNSSubdomain(project.Namespace) {
-		result = append(result, errors.NewFieldInvalid("Namespace", project.Namespace, "does not conform to lower-cased dns952"))
+	if len(project.Namespace) == 0 {
+		result = append(result, errors.NewFieldRequired("namespace", project.Namespace))
+	} else if !util.IsDNSSubdomain(project.Namespace) {
+		result = append(result, errors.NewFieldInvalid("namespace", project.Namespace, ""))
```

Comment:
Can you add "name must be a valid subdomain" message? We use in other places, so would be good to make it consistent.